Makefile

```diff
@@ -32,22 +32,43 @@ install-zsh-completions:
 install-fish-completions:
 	install -m644 -- contrib/completions.fish "$(FISHDIR)/exa.fish"
 
-
 uninstall:
-	-rm -- "$(DESTDIR)$(PREFIX)/share/man/man1/exa.1"
-	-rm -- "$(DESTDIR)$(PREFIX)/bin/exa"
-	-rm -- "$(BASHDIR)/exa"
-	-rm -- "$(ZSHDIR)/_exa"
-	-rm -- "$(FISHDIR)/exa.fish"
+	-rm -f -- "$(DESTDIR)$(PREFIX)/share/man/man1/exa.1"
+	-rm -f -- "$(DESTDIR)$(PREFIX)/bin/exa"
+	-rm -f -- "$(BASHDIR)/exa"
+	-rm -f -- "$(ZSHDIR)/_exa"
+	-rm -f -- "$(FISHDIR)/exa.fish"
 
 clean:
 	cargo clean
 
-
 preview-man:
-	nroff -man contrib/man/exa.1 | less
-
-
-.PHONY: all build install-exa install-man preview-man \
+	man contrib/man/exa.1
```

Comment:
I didn’t know you could do this! Just kind of stole [this answer from StackOverflow](http://stackoverflow.com/a/2658839/3484614).